db/seeds.rb

```diff
@@ -8,79 +8,359 @@
+  description: "Laboratorio de Mecatronica abierto a todas las carreras, util para crear e inovar.",
+  location: "Aulas 7 3er Piso",
+  remote_image_url: "http://lorempixel.com/600/440"
+)
+
+ls = lab.lab_spaces.create!(
+  name: "Sala de Herramientas",
+  description: "Nuestro interés está enfocado a la creación de una comunidad innovadora capaz de elaborar proyectos de impacto a largo plazo. Por eso, ofrecemos a la comunidad varias oportunidades de colaboración con el fin de generar valor no solamente a nuestra comunidad académica y organizaciones interesadas pero también, a todos stakeholders impactados.",
+  hours: "Lunes a Sabado 8 AM - 6:00 PM",
+  location: "Aulas 7 3er Piso",
+  contact_email: "mecatronica@itesm.mx",
+  contact_phone: "8111342200",
+  remote_image_url: "http://lorempixel.com/600/440"
+)
+
+eq = ls.equipment.create!(
+  name: "Mecatronica Escaner",
+  description: "Escáner 3D portable, capaz de realizar mediciones rápidas y confiables en 3D y a todo color.",
+  remote_image_url: "http://lorempixel.com/600/440",
+  technical_description: "Presición de 0.100 mm, área de escaneo de 143 x 108 mm, fuente de luz blanca y métodos de posición por geometría y/o color y/o objetivo.",
+  capability_ids: [Capabilities[4]],
+  material_ids: [Materials[3]]
+)
+
+eq.available_hours.create!(
+  day_of_week: 2,
+  start_time: "8:00 CT",
+  end_time: "18:00 CT"
+)
+
+eq = ls.equipment.create!(
+  name: "Impresora 3D 4000 Serie A",
+  description: "Materializa tus diseños 3D de manera fácil y precisa, creando piezas o maquetas volumétricas hechas a partir de tus modelos digitales.",
+  remote_image_url: "http://lorempixel.com/600/440",
+  technical_description: "Presición de 0.100 mm, área de escaneo de 143 x 108 mm, fuente de luz blanca y métodos de posición por geometría y/o color y/o objetivo.",
+  capability_ids: [Capabilities[2]],
+  material_ids: [Materials[3], Materials[4]]
+)
+
+eq.available_hours.create!(
+  day_of_week: 1,
+  start_time: "8:00 CT",
+  end_time: "22:30 CT"
+)
+
+eq.available_hours.create!(
+  day_of_week: 4,
+  start_time: "8:00 CT",
+  end_time: "22:30 CT"
+)
+
+eq = ls.equipment.create!(
+  name: "Mecatronica Cortadora láser",
+  description: "Graba, corta o marca tus diseño sobre múltiples materiales de forma precisa y rápida.",
+  remote_image_url: "http://lorempixel.com/600/440",
+  technical_description: "Cortadora Láser de tubo sellado de CO2 modelo STM-6090, área de trabajo de 60cm X 90cm y potencia de 80 watts.",
+  capability_ids: [Capabilities[3], Capabilities[4]],
+  material_ids: [Materials[3], Materials[4], Materials[5]]
+)
+
+eq.available_hours.create!(
+  day_of_week: 1,
+  start_time: "8:00 CT",
+  end_time: "17:00 CT"
+)
+
+eq.available_hours.create!(
+  day_of_week: 2,
+  start_time: "8:00 CT",
+  end_time: "17:00 CT"
+)
+
+eq.available_hours.create!(
+  day_of_week: 4,
+  start_time: "8:00 CT",
+  end_time: "14:00 CT"
+)
+
+## Seed users
+puts "\tSeeding users..."
+
+testu = User.create!(
```

Comment:
I say we change the name of this variable (something like `test_user` for example, we should avoid abbreviations of any kind). I know this is just the seeds file but let's make good habits.

Do we have admin user seeds? We should not remove those since there's a lot of testing that has to be done manually with admin accounts.

Reply:
The seed provides a test user and a superadmin, the superadmin can change the role of any user, admin and lab admin were not added in order not to have so much mock users.

Reply:
I think we should seed at least 1 of each role, but it's just a preference...